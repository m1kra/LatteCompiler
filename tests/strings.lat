

int main(){
    // This is a comment.
    printString("Hello there!");
    /*
    printString("Don't print this!");
    */
    printString("Test " + "string.");
    printString(repeat("abc ", 5));
    return 0;
}

string repeat(string s, int times){
    string r = "";
    int i = 0;
    while( i < times){
        r = r + s;
        i = i + 1;
    }
    return r;
}