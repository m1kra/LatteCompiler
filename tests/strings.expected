Hello there!
Test string.
abc abc abc abc abc 
